<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeteoRunwayWidget</class>
 <widget class="QWidget" name="MeteoRunwayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTop" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Runway:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxRunways"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonFilter">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>391</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetBottom" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="editRow">
        <property name="text">
         <string>Edit row</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonApply">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../share/resource.qrc">
          <normaloff>:/Images/OkIcon</normaloff>:/Images/OkIcon</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>355</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Last edit time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelEditTime">
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>comboBoxRunways</tabstop>
  <tabstop>toolButtonFilter</tabstop>
  <tabstop>toolButtonApply</tabstop>
  <tabstop>toolButtonReset</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../share/resource.qrc"/>
 </resources>
 <connections/>
</ui>
